### PR TITLE
Do not mutate browser source map object

### PR DESCRIPTION
**Pull request by kpdecker**
Fixes regression caused by #2478

### CODE REVIEW

File: server/build/webpack.js [hunk 1]
```diff
@@ -252,13 +252,14 @@ export default async function createCompiler (dir, { buildId, dev = false, quiet
         let output = transpiled.code
 
         if (map) {
-          map.sources = map.sources.map((source) => source.replace(/\?entry/, ''))
-          delete map.sourcesContent
+          let nodeMap = Object.assign({}, map)
```
Comment (timneutkens) [inline server/build/webpack.js]: I guess this can be just `const` since you're not reassigning the variable.

**Comment by frol** [comment]
I have just tested the fix and it works for me! @kpdecker Thank you!

**Review by timneutkens — APPROVED** [review]
Besides the const vs let comment everything's 💯 Thanks @kpdecker

**Comment by timneutkens** [comment]
@arunoda can you release this asap ❤️ Thanks!

**Pull request merged**